src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/SasTokenTest.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.reform.bulkscan.endtoendtests;
+
+import com.azure.storage.common.Utility;
+import com.azure.storage.common.implementation.StorageImplUtils;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.typesafe.config.ConfigFactory;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.apache.http.HttpHeaders;
+import org.junit.jupiter.api.Test;
+
+import java.time.OffsetDateTime;
+import java.util.Map;
+
+import static java.time.OffsetDateTime.now;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SasTokenTest {
+
+    private static String blobRouterUrl = ConfigFactory.load().getString("storage-account-url");
+
+    @Test
+    public void testSasToken() throws Exception {
+        assertThat(blobRouterUrl).isNotEmpty();
+
+        Response sasTokenResponse = RestAssured
+            .given()
+            .relaxedHTTPSValidation()
+            .baseUri(blobRouterUrl)
+            .header(HttpHeaders.CONTENT_TYPE, "application/json")
```

Comment:
there should be a constant for `"application/json"` somewhere

Reply:
I tried apache httpclient library but it doesn't have constant for "application/json".
In the other projects, we always used spring dependency

Reply:
ok